### PR TITLE
Set up Google Sans Flex GitHub Actions based build workflow

### LINKED ISSUE #1: Set up build workflow
We will move a static commit of @dberlow and @sannorozco's sources to the `build-the-build` branch so that we can begin the GH Actions remote build workflow work as development continues in the upstream repository.

Source state at repository transfer time: https://github.com/FontBureau/fb-GoogleSansFlex/commit/0d18f71aa89224910bdf32020cd70ec9f58f127e

Upstream FontBureau/fb-GoogleSansFlex branch: `latin-european-round`

@MariannaPaszkowska

### PR DESCRIPTION
**Pull request by chrissimpkins**
Closes #1

### CODE REVIEW

**Comment by chrissimpkins** [comment]
Updated sources with David and Santiago's latest updates in the upstream Font Bureau repository @ https://github.com/googlefonts/googlesans-flex/pull/2/commits/e0c3cb2b04130a35820ac8cce4366f865c663622

@belluzj @MariannaPaszkowska

**Comment by RickyDaMa** [comment]
Hi Chris,

We now have a working pipeline based on the Google Font Template! There's a few caveats / things to be aware of though:
1. The `source` folder as been renamed to `sources` to match the expectation of the Google font template. If we could keep the new UFOs or any changes to this that'd be appreciated
2. We don't build static OTFs. The curves in the UFOs are quadratic, so there's little point in building OTFs.
3. We currently can't build static TTFs, due to an error in removing overlaps when building. See the comments in `sources/config.yaml` for a more in-depth explanation
4. I've added one static instance to the GSFSquare designspace (so that we can try and build static fonts), however I daresay more are probably wanted

**Comment by chrissimpkins** [comment]
@MariannaPaszkowska @dberlow @sannorozco 

Thank you very much Ricky! Marianna and I will have a look at your documentation of the static build issue. We don't release GS families in otf format.  We won't need any otf format files.  That won't be a problem at all.  

Is the inability to build static instances a problem for you in the early Oct development schedule David / Santiago?

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+name: Build font and specimen
+
+on: [push, release]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (chrissimpkins) [inline .github/workflows/build.yaml:7]: We have larger GH Actions runner environments available if / when we need more parallelization and RAM support.  It is a simple configuration change on this `runs-on` line.  Swap out "ubuntu-latest" with one of the environment strings indicated with arrows in the review post (sorry, diff too big and GitHub UI won't allow me to drop image in this post)

**Review by chrissimpkins — COMMENTED** [review]
Larger GitHub Actions runner environment names if/when we need them

T7Fz7EPhqgNVApk

Example configuration from the googlefonts/googlesans repository: https://github.com/googlefonts/googlesans/blob/17f6141c95b742e41f3553bf60609ef8ca8b27dd/.github/workflows/cicd.yml#L23

**Comment by RickyDaMa** [comment]
Hey again Chris!

With some much-appreciated assistance from Nikolaus, the pipeline is all good to go. It can now build static TTFs (though this means we currently rely on our fork of `gftools` which provides the needed functionality, see https://github.com/googlefonts/gftools/pull/617 for the contribution upstream).

I've also switched from the provided designspaces to one I handcrafted to include all the masters and axes (dd7ef73 to 6f3055b). There's only one static instance defined at the moment though (Regular).

Nikolaus has also made some changes to the build script to make it more robust and to fix release tagging, which I'm informed now works correctly.

All in all, we think the pipeline is in pretty good shape now, so we'd appreciate a review from yourself!

**Comment by madig** [comment]
(Wait for new gftools release that incorporates https://github.com/googlefonts/gftools/pull/617 before merging)

**Comment by m4rc1e** [comment]
> With some much-appreciate assistance from Nikolaus, the pipeline is all good to go. It can now build static TTFs (though this means we currently rely on our fork of gftools which provides the needed functionality, see https://github.com/googlefonts/gftools/pull/617 for the contribution upstream).

Latest of gftools has this change, https://github.com/googlefonts/gftools/releases/tag/v0.9.19. Thanks for the PR!

**Comment by chrissimpkins** [comment]
Thank you very much Ricky and Nikolaus! Will take a look and be back in touch.  We plan to move 
@dberlow and @sannorozco over to this repository in the next week if they are ready.  I really appreciate all of your efforts with this!

**Comment by chrissimpkins** [comment]
I discussed the build workflow with David and Santiago today.  They really appreciate all of the efforts here.  The ability to automatically build fonts for testing will be useful to them as they begin their final phase of testing over the next few weeks.  I'll merge this to main, and they will have a look at the current VF build + designspace file definition over the upcoming week.  Going forward, Santiago will push updates to UFO + designspace definitions in a branch off of main and will open PRs for review before further changes land in the main branch.

They have some ideas about the auto-specimen generation that will be of use for reviewers (including internal Google creative direction teams) vs. what is currently provided in the zip archive.  We'll post new issues to address these things.

**Comment by chrissimpkins** [comment]
Making sure that we resolved all of Ricky's comments/questions:

> I've added one static instance to the GSFSquare designspace (so that we can try and build static fonts), however I daresay more are probably wanted

No worries at all.  Instance definitions require much more discussion and there is no decision on this yet.  No need to worry about additional statics for now.  So long as we can produce additional static ttf if/when needed.

> The source folder as been renamed to sources to match the expectation of the Google font template. If we could keep the new UFOs or any changes to this that'd be appreciated

@sannorozco please note the above comment about the sources directory name in the repository

Please let me know if you have any other questions that we haven't addressed.  Thanks again for all of the effort here!

File: .github/workflows/build.yaml (hunk 1 above)
Comment (chrissimpkins) [reply .github/workflows/build.yaml:7]: Addressed in #15

**Pull request merged**